### ISSUE TITLE
Tests are failing for go 1.8 and go 1.10.x

### ISSUE THREAD

**Issue by GabLeRoux** [cloudflare/tableflip#10]
I did not use Go a lot, but I found that tests are failing for go 1.8 and go 1.10.x.

go 1.8: https://travis-ci.com/GabLeRoux/tableflip/jobs/153029874

```
Using Go 1.5 Vendoring, not checking for Godeps
4.57s$ go get -t -v ./...
github.com/pkg/errors (download)
github.com/cloudflare/tableflip (download)
github.com/pkg/errors
github.com/GabLeRoux/tableflip
# github.com/GabLeRoux/tableflip
./fds.go:16: undefined: syscall.Conn
The command "eval go get -t -v ./... " failed. Retrying, 2 of 3.
github.com/GabLeRoux/tableflip
# github.com/GabLeRoux/tableflip
./fds.go:16: undefined: syscall.Conn
The command "eval go get -t -v ./... " failed. Retrying, 3 of 3.
github.com/GabLeRoux/tableflip
# github.com/GabLeRoux/tableflip
./fds.go:16: undefined: syscall.Conn
The command "eval go get -t -v ./... " failed 3 times.
The command "go get -t -v ./..." failed and exited with 2 during .
```

go 1.10.x: https://travis-ci.com/GabLeRoux/tableflip/jobs/153029876

```
--- FAIL: TestFdsListen (0.00s)
	fds_test.go:22: can't create new listener: listen unixgram : unknown network unixgram
```

I found this as part of #9 :v: I think it should be noted to the readme that this is only compatible with a few go versions.

**Comment by lmb** [comment]
Go 1.8 isn't supported, so the errors there don't surprise me :)

Go 1.10 seems weird. Is this running on a Linux box?

**Comment by GabLeRoux** [comment]
yep, on travis. I did not try locally and I did not go into much details as I was only traveling on the internets and I was curious about this project.

**Comment by lmb** [comment]
Fixed in #13
